test/TypeScriptGenerator-test.ts

```diff
@@ -1,16 +1,16 @@
-import {generateTestsFromFixtures} from 'relay-test-utils/lib/RelayModernTestUtils'
-import * as RelayTestSchema from 'relay-test-utils/lib/RelayTestSchema'
-import * as parseGraphQLText from 'relay-test-utils/lib/parseGraphQLText'
+import * as parseGraphQLText from 'relay-test-utils/lib/parseGraphQLText';
+import { generateTestsFromFixtures } from 'relay-test-utils/lib/RelayModernTestUtils';
+import * as RelayTestSchema from 'relay-test-utils/lib/RelayTestSchema';
 
-import {GraphQLCompilerContext, IRTransforms, transformASTSchema} from 'relay-compiler'
+import { GraphQLCompilerContext, IRTransforms, transformASTSchema } from 'relay-compiler';
 
-import * as TypeScriptGenerator from '../src/TypeScriptGenerator'
+import * as TypeScriptGenerator from '../src/TypeScriptGenerator';
 
 function generate(text, options) {
-  const schema = transformASTSchema(RelayTestSchema, [
-    IRTransforms.schemaExtensions[1], // RelayMatchTransform.SCHEMA_EXTENSION,
-    IRTransforms.schemaExtensions[2], // RelayRelayDirectiveTransform.SCHEMA_EXTENSION,
-    `
+	const schema = transformASTSchema(RelayTestSchema, [
+		IRTransforms.schemaExtensions[1], // RelayMatchTransform.SCHEMA_EXTENSION,
+		IRTransforms.schemaExtensions[2], // RelayRelayDirectiveTransform.SCHEMA_EXTENSION,
+		`
```

Comment:
Whitespace gone wild? 😄 

Perhaps run `yarn prettier`? I'm surprised the git hook didn’t automatically do this, though 🤔

Reply:
I'm quite certain I did exactly this, but let me try again.

Reply:
Ah, looks like `./test` isn’t included in the glob https://github.com/relay-tools/relay-compiler-language-typescript/blob/master/package.json#L38

Reply:
Right - I have changed this and included it in the PR as well.